OWNERS_ALIASES

```diff
@@ -1,18 +1,34 @@
-# See the OWNERS docs: https://git.k8s.io/community/contributors/devel/owners.md
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 aliases:
+  # active folks who can be contacted to perform admin-related
+  # tasks on the repo, or otherwise approve any PRS.
   controller-runtime-admins:
   - directxman12
   - droot
-  - pwittrock
-  controller-runtime-maintainers: []
+  - mengqiy
+
+  # non-admin folks who can approve any PRs in the repo
+  controller-runtime-approvers: []
+
+  # folks who can review and LGTM any PRs in the repo (doesn't
+  # include approvers & admins -- those count too via the OWNERS
+  # file)
   controller-runtime-reviewers:
   - alvaroaleman
```

Comment:
Should we add other folks here as well?

Reply:
this is just copied from the existing stuff

Reply:
sgtm